en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/subscription-validation.md

```diff
@@ -12,9 +12,7 @@ The following set of events are received by the Choreo Connect in order to perfo
 - Application key generation event (Generation of consumer key and secret)
 - Subscribing an API to application event.
 
-And also adapter pull the following details belonging to a particular tenant during the startup as well, in order to
-get the events that has happened before the starting of the gateway. Adapter will have list of environments assigned to it.
-Adapter will pull the APIs that are deployed in the specified set of environments only.
+And also adapter pull the following details belonging to a particular tenant during the startup as well, in order to get the events that has happened before the starting of the Gateway. Adapter will have list of environments assigned to it. Adapter will pull the APIs that are deployed in the specified set of environments only.
 
```

Comment:
```suggestion
In order to get the events that has happened before the Gateway startup, the adapter will pull the following details of the tenant defined for the Choreo Connect instance, during the startup. Adapter will have list of predefined environments from which it will pull the APIs and the other artifacts from. 

```